CHANGELOG.md

```diff
@@ -91,6 +91,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Raise an error if there are insufficient training batches when using a float value of `limit_train_batches` ([#12885](https://github.com/PyTorchLightning/pytorch-lightning/pull/12885))
 
 
+- The `WandbLogger` will now use the run name in the logs folder if it is provided, and otherwise the project name  ([#12157](https://github.com/PyTorchLightning/pytorch-lightning/issues/12157))
```

Comment:
```suggestion
- The `WandbLogger` will now use the run name in the logs folder if it is provided, and otherwise the project name ([#12157](https://github.com/PyTorchLightning/pytorch-lightning/issues/12157))
```